router/internal/edgerouter/config.go

```diff
@@ -112,6 +115,12 @@ func (config *Config) LoadConfigFromMap(configMap map[interface{}]interface{}) e
 			if flag, ok := flags["extend"]; ok {
 				config.ExtendEnrollment = flag.Value.String() == "true"
 			}
+
+			if flag, ok := flags["reenroll-seconds"]; ok {
```

Comment:
woops, meant to remove this. It was a debugging flat to force enrollment extension after x seconds